tsconfig.json

```diff
@@ -17,7 +17,8 @@
 			"DOM",
 			"ES5",
 			"ES6",
-			"ES7"
+			"ES7",
+			"ES2019"
```

Comment:
Please don't update this file for now. Let's find a way to accomplish the changes at the current API level please.